### ISSUE TITLE
How to make class.phpmailerpgp.php work?

### ISSUE THREAD

**Issue by sempiterna** [PHPMailer/PHPMailer#1833]
I'm trying to get the class.phpmailerpgp.php file from the openpgp branch (https://github.com/ravisorg/PHPMailer/blob/openpgp/class.phpmailerpgp.php) to work, but it's not showing what needs to be done.

The README.md claims I need to do this:

require 'PHPMailerAutoload.php';

But there is no PHPMailerAutoload.php file. I tried to place the file in the directory where the other phpmailer files are, but it seems it is not loaded with phpmailer. I also tried to add a "require" of that file, but that also does not work.

Is this openpgp branch still maintained?

**Comment by Synchro** [comment]
Indeed, that's the old autoloader for the 5.2 branch; 6.0 now relies on composer to deal with it, or you can load the classes manually from the `src` folder, as described in the readme. That branch was done by someone else (@ravisorg), not me, and there was discussion of it at the time in #505, but the 6.0 update didn't materialise. If someone feels like updating it, it could be merged.

**Comment by sempiterna** [comment]
I have installed PHPMailer using composer, so maybe that's why it doesn't work. Eventhough you have not done that branch, do you maybe know how to load that class when PHPMailer is installed using composer?

**Comment by Synchro** [comment]
No - that was the thing that needed finishing that was discussed in the ticket.

**Comment by sempiterna** [comment]
I reinstalled PHPMailer without composer, and after some changes (adding PHPMailer namespace to the class, load PHPMailerPGP, changing the last line of the class.phpmailerpgp.php because of exception errors, etc) I now arrived to a dead end.

It loads the class, but now complains that:

"PHP Fatal error:  Class 'PHPMailer\PHPMailer\gnupg' not found in"

It seems to me that it tries to call the PHP gnupg functions from within the PHPMailer namespace, where it doesnt exist. I don't know enough of namespaces to get any further unfortunately. The gnupg php extension is loaded and verified working in a test script.
